src/display-buffer.coffee

```diff
@@ -1314,6 +1154,25 @@ class DisplayBuffer extends Model
       # this one. Only emit when the marker still exists.
       @emitter.emit 'did-create-marker', marker
 
+  # TODO: serialize state in TextEditorElement, rather than saving scroll
+  # positions here.
```

Comment:
> but maybe it's something that is only assigned by the view and read one time when it is attached to the DOM.

:+1: for this. Ideally, I'd :heart: the view to have a serialized state of its own because of the reasons pointed out in https://github.com/atom/atom/pull/8905#discussion_r40288745, but I am :cool: with keeping the model up to date with the current logical coordinates as well.